content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
 
-To retrieve a single list in chunks, two new parameters `limit` and `continue` are supported on collection requests and a new field `continue` is returned from all list operations in the list `metadata` field. A client should specify the maximum results they wish to receive in each chunk with `limit` and the server will return up to `limit` resources in the result and include a `continue` value if there are more resources in the collection. The client can then pass this `continue` value to the server on the next request to instruct the server to return the next chunk of results. By continuing until the server returns an empty `continue` value the client can consume the full set of results.
+To retrieve a single list in chunks, two parameters `limit` and `continue` are supported on collection requests. When a List request specifies a `limit`, a field `continue` is returned in the `metadata` field. A client should specify the maximum results they wish to receive in each chunk with `limit`; the API server returns up to `limit` resources in the result. If there are more resources in the collection, the API server response also includes a `continue` field. The client can then pass this `continue` value to the API server in the next request, instructing the API server to return the next chunk of results. By continuing until the server returns an empty `continue` value, the client can consume the full set of results.
 
-Like a watch operation, a `continue` token will expire after a short amount of time (by default 5 minutes) and return a `410 Gone` if more results cannot be returned. In this case, the client will need to start from the beginning or omit the `limit` parameter.
+Like a Watch operation, a `continue` token expires after a defined amount of time (5 minutes by default). If more results cannot be returned the API server returns a 410 (Gone) status. On receiving a 410 (Gone) response, the client needs to make a new request (possibly omitting the `limit` parameter, or using a larger `limit`).
 
-For example, if there are 1,253 pods on the cluster and the client wants to receive chunks of 500 pods at a time, they would request those chunks as follows:
+For example, if there are 1,253 pods on the cluster and the client wants to receive chunks of 500 pods at a time, the client can request those chunks as follows:
 
-1. List all of the pods on a cluster, retrieving up to 500 pods each time.
+1. List all of the Pods on a cluster, retrieving up to 500 Pods each time.
```

Comment:
Possible edit?
List all pods in a cluster, retrieving up to 500 pods each time.

Reply:
In the _API_, you're really dealing with Pods.
Maybe:
```suggestion
1. List all of the `Pods` on a cluster, retrieving up to 500 objects each time.
```

Reply:
I've omitted this change from the latest revision.